packages/coordinator/src/screens/AuthScreens/ResetPasswordScreen.tsx

```diff
@@ -1,33 +1,75 @@
 import { useState } from "react";
 import styles from "./AuthScreens.module.scss";
 import Input from "../../components/Input";
-import Button from "../../components/Button";
+import Button, { ButtonVariant } from "../../components/Button";
 import logoImage from "./../../assets/blood-bank-zichron-logo.svg";
+import popupImage from "../../assets/popup_resetpass.png";
 import chevronSvg from "./../../assets/icons/chevron-right-small.svg";
 import { useNavigate } from "react-router-dom";
+import { Popup } from "../../components/Popup/Popup";
 
 export interface ResetPasswordScreenProps {
   onResetPassword: (
     email: string,
     emailError: (error: string) => void
-  ) => Promise<void>;
+  ) => Promise<boolean>;
 }
 
 export default function ResetPasswordScreen(props: ResetPasswordScreenProps) {
   const [email, setEmail] = useState("");
   const [emailError, setEmailError] = useState("");
   const [isLoading, setIsLoading] = useState(false);
 
+  const [popupOpen, setPopupOpen] = useState<boolean>(false);
+  const [popupError, setPopupError] = useState<string | undefined>(undefined);
+
   const navigate = useNavigate();
 
   const resetPassword = async () => {
     setIsLoading(true);
-    await props.onResetPassword(email, setEmailError);
+    const res = await props.onResetPassword(email, setEmailError);
     setIsLoading(false);
+
+    return res;
+  };
+
+  const handleResetPass = async () => {
+    const resetStatus = await resetPassword();
+    setPopupOpen(true);
+
+    if (!resetStatus) {
+      setPopupError("כתובת המייל שלך לא נמצאה");
+    }
   };
 
+  const popupContent = (
+    <div className={styles.resetPassPopupContent}>
+      <img src={popupImage} alt="" className={styles.resetPassPopupImg} />
```

Comment:
`Popup` has a prop `image` - use it

Reply:
It also has buttons. Use them instead of creating ones here

Reply:
1. There is no image prop in popup now.
2. I cant change the props of the buttons in the popup component.

Reply:
If you need something that is not exposed in `Popup` - expose it